reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -755,6 +755,9 @@
             { "Key": "Name",
               "Value": {"Fn::Join": [".", ["ose-master01",{"Ref": "PublicHostedZone"}]]}
             },
+            { "Key": "KubernetesCluster",
```

Comment:
Since this key is being added to all of the hosts, instead of setting it on each host, you can set it for the entire stack using the `tags` param with the cloudformation module